diff --git a/node_modules/modular-scripts/dist-cjs/esbuild-scripts/config/createEsbuildConfig.js b/node_modules/modular-scripts/dist-cjs/esbuild-scripts/config/createEsbuildConfig.js
index c4cc557..10ec336 100644
--- a/node_modules/modular-scripts/dist-cjs/esbuild-scripts/config/createEsbuildConfig.js
+++ b/node_modules/modular-scripts/dist-cjs/esbuild-scripts/config/createEsbuildConfig.js
@@ -55,8 +55,8 @@ function createEsbuildConfig(paths, config = {}) {
       '.png': 'file',
       '.webp': 'file',
       // font file format loaders
-      '.woff': 'file',
-      '.ttf': 'file',
+      '.woff': 'dataurl',
+      '.ttf': 'dataurl',
       // enable JSX in js files
       '.js': 'jsx'
     },
diff --git a/node_modules/modular-scripts/react-scripts/config/parts/baseConfig.js b/node_modules/modular-scripts/react-scripts/config/parts/baseConfig.js
index 7ce9194..c9c53d8 100644
--- a/node_modules/modular-scripts/react-scripts/config/parts/baseConfig.js
+++ b/node_modules/modular-scripts/react-scripts/config/parts/baseConfig.js
@@ -288,6 +288,10 @@ function createConfig({
                 shouldUseSourceMap,
               }),
             },
+            {
+              test: /\.(md|txt)$/,
+              use: require.resolve('raw-loader'),
+            },
             // "file" loader makes sure those assets get served by WebpackDevServer.
             // When you `import` an asset, you get its (virtual) filename.
             // In production, they would get copied to the `build` folder.
